src/Queue.php

```diff
@@ -84,7 +84,7 @@ class Queue implements \Countable
      * The `$handler` parameter must be a valid callable that accepts your job
      * parameters, invokes the appropriate operation and returns a Promise as a
      * placeholder for its future result. If the given argument is not a valid
-     * callable, this method will reject with an `InvalidArgumentExceptionn`
+     * callable, this method will reject with an `InvalidArgumentException`
```

Comment:
LGTM, the README.md has the exact same typo, can you amend this? :shipit:

Reply:
Done, my `git` skills are still too poor to do a real "amend", so you might need to squash he commits into one....